libcloud/dns/drivers/cloudflare.py

```diff
@@ -415,6 +434,36 @@ def ex_enable_ipv6_support(self, zone):
     def ex_disable_ipv6_support(self, zone):
         raise NotImplementedError('not yet implemented in v4 driver')
 
+    def _normalize_record_data_for_api(self, type, data):
+        """
+        Normalize record data for "special" records such as CAA so it can be
+        used with the CloudFlare API.
+        """
+        if not data:
+            return data
+
+        if type == RecordType.CAA:
+            # Replace whitespace with \t character which CloudFlare API
+            # expects
+            data = data.replace(' ', '\t')
```

Comment:
I decided to have CloudFlare driver use whitespace as a separator since it appears most of the providers use that and not tab which is used by CloudFlare.

This way it will be consistent format across all the drivers.